docs/usage.md

```diff
@@ -1,103 +1,152 @@
 
-  - The playbooks are by default configured to execute in "Test Mode"
-  - Under the "Test Mode", all the email communication to be towards DPO, Affected Individuals or DPA will be done using the test email ID that is configured for testing while in 'Production Mode' the actual email IDs will be used
-  - In order to change execution mode or test email-ID, follow the below procedure:
-    - Goto to the "Create Data Compliance Record" playbook under the '10 - SP - GDPR Framework' collection and open the 'Configuration' step
-    ![Execution Mode Configuration](res/execution-mode-configuration.png)
-    - To change of playbooks execution mode from "Test Mode" to "Production", Modify value of **testMode** variable to `false`
-    - Likewise, to change test email-ID change value of **testEmail** variable
-    ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parametes.png)
+- The playbooks are by default configured to execute in *Test Mode*
+  - The *Test Mode* uses a test email address for all email communication towards Data Protection Officer (DPO), affected individuals, or Data Protection Authority(DPA)
+  - The *Production Mode* uses actual email addresses  of relevant authorities and stakeholders
 
+### Production Mode
 
-## Gathering Details on Personal Data Breach
+To change the execution mode to *Production* make the following changes:
 
-- When a personal data breach is detected in an incident, choose the value of the **Was Personal Data Affected?** field as **Yes**.
+1. Go to the *Create Data Compliance Record* playbook under the **10 - SP - GDPR Framework** collection and open the *Configuration* step
 
-    ![Personal Data Affected](res/personal-data-affected.png)
+  ![Execution Mode Configuration](res/execution-mode-configuration.png)
 
+2. Modify the value of the `testMode` variable to `false`
+3. Change the value of the `testEmail` variable to a valid email address.
 
-- A pop-up appears that collects additional information related to the incident
+  ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parameters.png)
 
-  **Note:-** When playbooks executed in 'Test Mode', the GREEN icon shown in below screenshot will appear in all the task in this solution pack
+As a failsafe, when the *Production* mode is set we append `_demo` while fetching DPA's email as shown in the following image. This action is to prevent any accidental sending of mail to the Data Protection Authority (DPA).
 
-    ![Personal Data Additional Details](res/personal-data-additional-details.png)
+![DPA Email Append](res/dpa-appended-email.png)
 
+It is recommended that in `Production Mode` before you execute **Provide DPO and DPA Contact** playbooks, make the following changes
+1. Go to **Provide DPO and DPA Contact** playbook
+2. Open the *Get DPA Contact* step and remove `_demo` from `dpaEmail` variable as shown
+    ![Provide DPO and DPA Contact Changes](res/provide-dpo-and-dpa-contact.png)
```

Comment:
Broken screenshot link
![image](https://user-images.githubusercontent.com/24773177/194795138-ca48f835-94e3-47f3-aea5-bb77d04d0179.png)

Reply:
At multiple places the images have broken link.